servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/DefaultExecutor.java

```diff
@@ -127,23 +127,16 @@ public void accept(final Runnable runnable) {
     }
 
     /**
-     * {@link AutoCloseable} interface will invoke {@link ExecutorService#shutdown()}.
+     * {@link Runnable} interface will invoke {@link ExecutorService#shutdown()}.
      */
-    private interface InternalExecutor extends Function<Runnable, Cancellable>, AutoCloseable {
-
-        @Override
-        void close();
+    private interface InternalExecutor extends Function<Runnable, Cancellable>, Runnable {
```

Comment:
doesn't this revert a change you recently make?

Reply:
Yes, weird. I had not noticed this strange revert.

Reply:
Yes, very strange. I had completely missed that it reverted the other change.